ci/code_checks.sh

```diff
@@ -122,7 +122,7 @@ if [[ -z "$CHECK" || "$CHECK" == "doctests" ]]; then
       pandas/io/sas/ \
       pandas/io/sql.py \
       pandas/tseries/ \
-      pandas/io/formats/style_render.py
+      pandas/io/formats/format.py
```

Comment:
Sorry, didn't see this earlier. Why are you replacing the previously validated file? You can check of the whole `pandas/io/formats` directory is passing the tests, and validate the whole directory, but of it's not, add your file but leave the other file too.

Reply:
I think that was just me copying the file path into there and accidently pasting it over the last line sorry!